hal/src/thumbv6m/usb/bus.rs

```diff
@@ -995,6 +997,7 @@ impl Inner {
                 }
                 Err(err) => {
                     dbgprint!("UsbBus::read from ep {:?} -> {:?}\n", ep, err);
+                    drop(bank); // bank holds a reference to self.desc, which print_epstatus() needs
```

Comment:
> Just noticed this might've broken isochronous transfers; I've got some other local changes that might also be where the problem comes from, but need to do some more digging.

This was the problem - previously this PR just deleted the `drop(bank);` above.